CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ requirements:
 - [Convert equipment to a base field #956](https://github.com/farmOS/farmOS/pull/956)
 - [Convert quick to a base field #957](https://github.com/farmOS/farmOS/pull/957)
 - [Make farmOS API OAuth2 Server optional #973](https://github.com/farmOS/farmOS/pull/973)
+- [Move QuantityCsvNormalizer to farm_csv module #977](https://github.com/farmOS/farmOS/pull/968)
```

Comment:
The commit and this line in changelog still reference PR 968

Reply:
Oops! Good catch @paul121. Fixed.